Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Mesh/MeshFeatureProcessorInterface.h

```diff
@@ -40,6 +40,33 @@ namespace AZ
             AZ::ConsoleFunctorFlags::Null,
             "Enable instanced draw calls in the MeshFeatureProcessor.");
 
+        AZ_CVAR(
+            bool,
+            r_meshInstancingEnabledForTransparentObjects,
+            false,
+            nullptr,
+            AZ::ConsoleFunctorFlags::Null,
+            "Enable instanced draw calls for transparent objects in the MeshFeatureProcessor. Use this only if you have many instances of the same "
+            "transparent object, but don't have multiple different transparent objects mixed together. See documentation for details.");
+
+        AZ_CVAR(
+            size_t,
+            r_meshInstancingBucketSortScatterBatchSize,
+            512,
+            nullptr,
+            AZ::ConsoleFunctorFlags::Null,
+            "Batch size for the first stage of the mesh instancing bucket sort. "
+            "Can be modified to find optimal load balancing for the multi-threaded tasks.");
+
+        AZ_CVAR(
+            bool,
+            r_meshInstancingDebugForceUniqueObjectsForProfiling,
+            false,
+            nullptr,
+            AZ::ConsoleFunctorFlags::Null,
+            "Enable instanced draw calls in the MeshFeatureProcessor, but force one object per draw call. "
+            "This is helpful for simulating the worst case scenario for instancing for profiling performance.");
+
```

Comment:
If you define these cars using AZ_CVAR_EXTERNED in the header file, you can move this code to the .cpp file and can just remove all the code related to console->PerformCommand from the cpp file as well. I did something similar with r_gpuMarkersMergeGroups that can be used as an example